axelrod/strategies/lookerup.py

```diff
@@ -165,23 +167,310 @@ def strategy(self, opponent):
         return action
 
 
-# Create several classes at runtime based on the data in the look up tables
-# loaded above, one for each key.
-
-for k, (initial, pattern) in patterns.items():
-    name, plays, op_plays, op_start_plays = k
-    table = create_lookup_table_from_pattern(
-        plays, op_plays, op_start_plays, pattern)
-    class_name = "EvolvedLookerUp{}{}_{}_{}".format(
-        name, plays, op_plays, op_start_plays)
-    # Dynamically create the class
-    new_class = type(class_name, (LookerUp,), {})
-    new_class.__init__ = init_args(partial(
-        new_class.__init__, lookup_table=table, initial_actions=initial))
-    new_class.name = class_name
-    # Add the generated class to the module dictionary so it can be
-    # imported into other modules
-    setattr(module, class_name, new_class)
+class EvolvedLookerUp0_0_1(LookerUp):
```

Comment:
I don't know that we actually want to add all of these to the library. I only included a few in the axelrod.strategies list -- maybe just add those? Some of the small ones shadow strategies in the library already, and the ones with larger parameters don't seem to outperform 2_2_2 for the most part.